tasks/phpstan/phpstan-baseline.neon

```diff
@@ -243,3 +243,8 @@ parameters:
 			message: "#^Parameter \\#1 \\$newsletter of method MailPoet\\\\Entities\\\\NewsletterTemplateEntity\\:\\:setNewsletter\\(\\) expects MailPoet\\\\Entities\\\\NewsletterEntity\\|null, object given\\.$#"
 			count: 1
 			path: ../../lib/NewsletterTemplates/NewsletterTemplatesRepository.php
+
+		-
+			message: "#^Parameter \\#2 \\$segment of class MailPoet\\\\Entities\\\\NewsletterSegmentEntity constructor expects MailPoet\\\\Entities\\\\SegmentEntity, object given\\.$#"
```

Comment:
Yes. The Doctrine extension for PHPStan is not working in our case and even though I have a hacking branch with prefixing it, I'm not sure if we can safely use it because we're using a pretty old Doctrine version and I guess the extension has stubs for current versions. We can also create our own stubs I guess (this is now writable using generics).